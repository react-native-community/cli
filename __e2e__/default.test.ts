import {
  runCLI,
  getTempDirectory,
  cleanupSync,
  writeFiles,
} from '../jest/helpers';

const DIR = getTempDirectory('test_default_behavior');

beforeEach(() => {
  cleanupSync(DIR);
  writeFiles(DIR, {});
});
afterEach(() => {
  cleanupSync(DIR);
});

test('shows up help information without passing in any args', () => {
  const {stdout} = runCLI(DIR);
  expect(stdout).toMatchSnapshot();
});
