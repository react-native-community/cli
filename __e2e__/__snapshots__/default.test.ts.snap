// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows up help information without passing in any args 1`] = `
"Usage: bin <command> [options]

Options:
  --version                     Print CLI version
  --verbose                     Increase logging verbosity
  -h, --help                    output usage information

Commands:
  init [options] <projectName>  Initialize a new React Native project named <projectName> in a directory of the same name.
  doctor [options]              [EXPERIMENTAL] Diagnose and fix common Node.js, iOS, Android & React Native issues."
`;
